.gitmodules

```diff
@@ -22,3 +22,6 @@
 [submodule "lib/AWS/defender"]
 	path = lib/AWS/defender
 	url = https://github.com/aws/Device-Defender-for-AWS-IoT-embedded-sdk.git
+[submodule "lib/FreeRTOS/coreMQTT-Agent"]
```

Comment:
Should lines 7-9 be deleted from this file?

Reply:
Good catch. Yes.